python/Create_Large_Test_Files.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Creating Large HMDA Submission Test Files
+# 
+# This notebook demonstrates a class for creating large HMDA submission test files.
+
+# In[ ]:
+
+
+#The following code imports the required packages.
+import math
+import json
+import os
+import pandas as pd
+import random
+import string
+import time
+import yaml
+import shutil as sh
+
+from lar_generator import lar_gen #Imports lar_gen class. 
+lg = lar_gen() #Instantiates lar_gen class as lg. 
+
+
+# ### Creating Clean Test Files
+# 
+# The following defines a function that creates clean test files. It uses an existing clean file to create a new clean file of any row count. This set of functions is from the Clean_Files class defined below. An existing clean test file can be separated into its Transmittal Sheet and LAR components, using the Clean_Files class function separate_lar_ts. 
+
+# In[ ]:
+
+
+cf = Clean_Files()
+cf.separate_lar_ts(file = "../edits_files/file_parts/clean_file_1000_rows_Bank1.txt")
+
+
+# ### Creating Clean Test Files With Specified Counts in a List
+# A loop may be used to create clean large test files with the counts in a list. 
+
+# In[ ]:
+
+
+rows = [297]
+
+for row in rows:
+    ts_file = "../edits_files/file_parts/ts_data.txt" #Stores the file path for TS data as ts_file. 
+    lar_file = "../edits_files/file_parts/lar_data.txt" #Stores the file path for LAR data as lar_file.
+    df = cf.get_rows(lar_file = lar_file, row_count = row)
+    df = cf.assign_uli(df=df, ts_file = ts_file)
+    cf.lar_to_hmda_file(ts_file=ts_file, lar_rows_df=df)
+
+
+# In[ ]:
+
+
+class Clean_Files(object):
+    def separate_lar_ts(self, file=None):  
+        """
+         Separates a clean file into Transmittal Sheet (TS) and Loan Application Register (LAR) data.
+         Saves the LAR data and TS data as text files in the /edits_files/file_parts/ directory. 
+        
+        """
+        with open(file, 'r+' ) as f: #Opening the submission file. 
+            ts_row = f.readline() #Reading in the first row as the TS submission. 
+            lar_rows = f.readlines() #Reading in the other rows as the LAR submission.  
+        with open("../edits_files/file_parts/lar_data.txt", 'w') as out_lar: #Writes a file of LAR data. 
+            out_lar.writelines(lar_rows)
+        with open("../edits_files/file_parts/ts_data.txt", 'w') as out_ts: #Writes a file of TS data. 
+            out_ts.writelines(ts_row)
+        
+        #Prints the location of each data file. 
+        statement = """
+           LAR file saved as: "../edits_files/file_parts/lar_data.txt", 
+           TS  file saved as: "../edits_files/file_parts/ts_data.txt" 
+           """
+        
+        print(statement)
+        return
```

Comment:
The return command here is just giving back None. Consider removing it or modifying it to have some value as to not confuse users expecting the function to return something.